common/src/app/common/files/changes.cljc

```diff
@@ -366,9 +366,33 @@
      [:type [:= :del-typography]]
      [:id ::sm/uuid]]]
 
-   [:update-active-token-themes
-    [:map {:title "UpdateActiveTokenThemes"}
-     [:type [:= :update-active-token-themes]]
+   [:set-tokens-lib
+    [:map {:title "SetTokensLib"}
+     [:type [:= :set-tokens-lib]]
+     [:tokens-lib ::sm/any]]]  ;; TODO: we should define a plain object schema for tokens-lib
```

Comment:
This is the only change definition that is not clear to me. I mean, this change is too big, and any use of it will overwrite any more granular action on tokens that other concurrent user is doing.

I think we should not have this method at all... is this used for anything relevant?

Reply:
This is already being used in the file import, where the whole library is replaced by a new one read from a json file. Anyway this is going to be revised in depth during the next work with the synchronization mechanisms, so I suppose we can leave it as is for now.

But don't add an entry point in the plugins api for this for now.